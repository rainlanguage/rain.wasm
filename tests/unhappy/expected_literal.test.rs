#[macro_use]
extern crate wasm_bindgen_utils;

pub struct A;
impl_custom_tsify!(A);

fn main() {}
