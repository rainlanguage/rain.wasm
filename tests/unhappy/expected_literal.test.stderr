error: unexpected end of macro invocation
 --> tests/unhappy/expected_literal.test.rs:5:21
  |
5 | impl_custom_tsify!(A);
  |                     ^ missing tokens in macro arguments
  |
note: while trying to match `,`
 --> src/macros.rs
  |
  |     ($type_name:ident $(< $($generics:ident),+ >)?, $decl:literal) => {
  |                                                   ^
